waku/node/v2/wakunode2.nim

```diff
@@ -223,6 +182,56 @@ proc info*(node: WakuNode): WakuInfo =
   let wakuInfo = WakuInfo(listenStr: listenStr)
   return wakuInfo
 
+proc mountFilter*(node: WakuNode) =
+  info "mounting filter"
+  proc filterHandler(requestId: string, msg: MessagePush) {.gcsafe.} =
+    info "push received"
+    for message in msg.messages:
+      node.filters.notify(message, requestId)
+
+  node.wakuFilter = WakuFilter.init(node.switch, node.rng, filterHandler)
+  node.switch.mount(node.wakuFilter)
+  node.subscriptions.subscribe(WakuFilterCodec, node.wakuFilter.subscription())
+
+proc mountStore*(node: WakuNode) =
+  info "mounting store"
+  node.wakuStore = WakuStore.init(node.switch, node.rng)
+  node.switch.mount(node.wakuStore)
+  node.subscriptions.subscribe(WakuStoreCodec, node.wakuStore.subscription())
+
+proc mountRelay*(node: WakuNode, topics: seq[string] = newSeq[string]()) {.async, gcsafe.} =
```

Comment:
Out of curiosity, does it work when not mounting relay?

Reply:
It does not seem to work right now, I can investigate further why not.

Reply:
@oskarth so I started 3 nodes:
 A) One with Relay, Filter and Store
 B) One with Relay
 C) One with Store

I sent a message from B, it was received by A, I then queried from C and received the expected message. So I would say it does work.